src/main/java/de/timeline/TimelineController.java

```diff
@@ -30,8 +30,10 @@ public void createNewEvent(@RequestBody Event event) {
 		timelineService.createEvent(event);
 	}
 
-	@DeleteMapping("/events")
-	public ResponseEntity deleteOldEvent(@PathVariable UUID uuid) {
+	@DeleteMapping("/events/{uuid}")
+	// hier fehlte die uuid noch als Platzhalter im Mapping, das war sicher im
+	// Video so gezeigt!
```

Comment:
Ja es kamen vermutlich zwei Probleme zusammen. Immer nochmal die drei Request-Konstrukte im Kopf behalten... URL, Request-Body, Request-Header. Einen Body gab es nicht mehr, deswegen musstest Du die uuid irgendwie in der URL unterbringen und dementsprechend auch die Tests umbauen.